Wrappers/Python/test/test_dataexample.py

```diff
@@ -157,116 +156,74 @@ def test_load_SIMULATED_CONE_BEAM_DATA(self):
 class TestRemoteData(unittest.TestCase):
 
     def setUp(self):
-        
         self.data_list = ['WALNUT','USB','KORN','SANDSTONE']
-        self.shapes_path = os.path.join(dataexample.CILDATA.data_dir, dataexample.TestData.SHAPES)
 
-    def mock_urlopen(self, mock_urlopen, zipped_bytes):
-        mock_response = MagicMock()
-        mock_response.read.return_value = zipped_bytes
-        mock_response.__enter__.return_value = mock_response
-        mock_urlopen.return_value = mock_response
 
-    @unittest.skipIf(platform.system() == 'Windows', "Skip on Windows")   
-    @patch('cil.utilities.dataexample.urlopen')
-    def test_unzip_remote_data(self, mock_urlopen):
-        '''
-        Test the _download_and_extract_data_from_url function correctly extracts files from a byte string
-        The zipped byte string is mocked using a temporary local zip file
+    def mock_zenodo_get(*args):
```

Comment:
Possibly for each data we could also test that zenodo_get is being called with the correct zenodo record / zip file combo, but also that might be overkill 😄

Reply:
Good points, I have added test_zenodo_record_exists which tries to access the record and prints the available zip files, it will fail if the record can't be accessed or the zip file we want is not in the list. I think this addresses both points because it will fail if the record / zip file combo is wrong or not available

Reply:
Excellent 😄